packages/apps/spaces/app/organizations/[id]/components/Tabs/panels/AccountPanel/RenewalLikelihood/RenewalLikelihood.tsx

```diff
@@ -0,0 +1,149 @@
+'use client';
+
+import { Flex } from '@ui/layout/Flex';
+import { Heading } from '@ui/typography/Heading';
+import { Text } from '@ui/typography/Text';
+import { IconButton } from '@ui/form/IconButton';
+import { Icons, FeaturedIcon } from '@ui/media/Icon';
+import { Divider } from '@ui/presentation/Divider';
+import { Card, CardBody, CardFooter } from '@ui/presentation/Card';
+import { useDisclosure } from '@ui/utils';
+import { InfoDialog } from '@ui/overlay/AlertDialog/InfoDialog';
+
+import {
+  Likelihood,
+  RenewalLikelihoodModal,
+  Value,
+} from './RenewalLikelihoodModal';
+
+interface RenewalLikelihoodProps {
+  value: Value;
+  onChange: (value: Value) => void;
+}
+
+export const RenewalLikelihood = ({
+  value,
+  onChange,
+}: RenewalLikelihoodProps) => {
+  const update = useDisclosure();
+  const info = useDisclosure();
+  const { likelihood, reason } = value;
```

Comment:
💅 that can be destructurized in line 25

Reply:
I rather not since I'm passing the entire object as `value={value}` at line 89.